ISL.ReIdentification.Core/Services/Foundations/DelegatedAccesses/DelegatedAccessService.cs

```diff
@@ -32,6 +33,12 @@ public ValueTask<DelegatedAccess> AddDelegatedAccessAsync(DelegatedAccess delega
                 return await this.ReIdentificationStorageBroker.InsertDelegatedAccessAsync(delegatedAccess);
             });
 
+        public ValueTask<IQueryable<DelegatedAccess>> RetrieveAllDelegatedAccessesAsync() =>
+            TryCatch(async () =>
+            {
+                return await this.ReIdentificationStorageBroker.SelectAllDelegatedAccessesAsync();
+            });
```

Comment:
We can simplify this since it is only a single task and this task is already awaited in the TryCatch

```suggestion
        public ValueTask<IQueryable<DelegatedAccess>> RetrieveAllDelegatedAccessesAsync() =>
            TryCatch(this.ReIdentificationStorageBroker.SelectAllDelegatedAccessesAsync());
```

Reply:
The above syntax doesn't seem to work, however I have condensed to a single line removing the curly braces and return

Reply:
```cs
        public ValueTask<IQueryable<DelegatedAccess>> RetrieveAllDelegatedAccessesAsync() =>
            TryCatch(this.ReIdentificationStorageBroker.SelectAllDelegatedAccessesAsync());
```
This should be fine.  VS should also suggest it as an optimization